google-cloud-bigquery/src/main/java/com/google/cloud/bigquery/testing/package-info.java

```diff
@@ -17,7 +17,12 @@
 /**
  * A testing helper for Google BigQuery.
  *
- * <p>A simple usage example:
+ * <p>A simple usage example: 1. Create a test Google Cloud project.
+ *
+ * <p>2. Download a JSON service account credentials file from the Google Developer's Console.
+ *
+ * <p>3. Create a RemoteBigQueryHelper object using your project ID and JSON key. Here is an example
+ * that uses the RemoteBigQueryHelper to create a dataset. 4. Run tests.
```

Comment:
"4. Run tests." should be a new paragraph.

Reply:
Fixed.